sdk/inc/azure/core/az_http.h

```diff
@@ -114,10 +117,10 @@ typedef enum
  */
 typedef struct
 {
-  int16_t max_retries;
+  az_http_status_code const* status_codes;
```

Comment:
From the API review:
> Is the const here in the right place? The status codes should not change.

Reply:
Yes, it is. It should't be in the other place - it would be the same as if `max_retries` was declared as `int const max_retries`.